content/blog/2021-05-03-introducing-prometheus-conformance-program.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Introducing the Prometheus Conformance Program
+created_at: 2021-05-03
+kind: article
+author_name: Richard "RichiH" Hartmann
+---
+
+Prometheus is the standard for metric monitoring in the cloud native space and beyond. To ensure interoperability, to protect users from suprises, and to enable more parallel innovation, the Prometheus project is introducing the [Prometheus Conformance Program](https://github.com/cncf/prometheus-conformance) with the help of [CNCF]{https://www.cncf.io/} to certify component compliance and Prometheus compatibility.
+
+The CNCF Governing Board is expected to formally start the program during their next meeting. We invite the wider community to help improve our test in this ramp-up phase.
```

Comment:
"formally review and approve the program during..."

Reply:
Fixed, thanks!